src/main/scala/edu/berkeley/ce/rockslicing/NumericUtils.scala

```diff
@@ -5,7 +5,7 @@ import scala.math.BigDecimal
  * Numerical utilities for implementation and test cases.
  */
 object NumericUtils {
-  val EPSILON = 1.0e-6
+  val EPSILON = 1.0e-5
```

Comment:
Hmm, I guess I could see either choice working. Modifying `approximateEquals` seems better, but then we're using a different `EPSILON` just in that one place...

I'd say go ahead and modify `approximateEquals`, but I'll leave it up to you.